app/models.py

```diff
@@ -2071,3 +2070,13 @@ def serialize(self):
             "created_at": self.created_at.strftime(DATETIME_FORMAT),
             "updated_at": self.updated_at.strftime(DATETIME_FORMAT) if self.updated_at else None,
         }
+
+
+class ReturnedLetter(db.Model):
+    __tablename__ = 'returned_letters'
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    reported_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
```

Comment:
Is there any chance that we will populate the table twice on the same day? If there is, then having this column as a date, not a datetime, would make things easier because we are only interested in the daily results, so we wouldn't have to then aggregate the results by day.